src/SignalR/clients/csharp/Client.Core/src/HubConnection.cs

```diff
@@ -985,11 +1010,17 @@ private async Task ReceiveLoop(ConnectionState connectionState)
             }
         }
 
+        /// <summary>
+        /// This method is for internal framework use and should not be called by user code.
```

Comment:
Or just make it private :)

Reply:
I didn't want to bog this PR down with that, but I can just remove the doc comment. It was public in 2.2 so our standard policy is to deprecate it over one release first. See https://github.com/aspnet/AspNetCore/issues/7977 for more. I'll remove these changes and we can just discuss things there.